contents/calendar/_index/main.markdown

```diff
@@ -25,6 +25,21 @@ events (excluding HackerspaceSG’s cleaning every Tuesday, of course).
 
 # Hosting Events
 
+## COVID-19 measures
+
+In view of the current outbreak of the Coronavirus Disease 2019 (COVID-19), HackerspaceSG strongly encourages event organisers to take the following measures:
+
+* Ask visitors to fill out a travel and health declaration form, and to include their personal information for contact tracing purposes, before the event.
```

Comment:
do we need to add a warning that false declaration may have legal consequences here in Singapore? the crowd that some events attract may have a higher aversion to giving out private information, be it due to differences in laws, culture, etc. see: https://www.tnp.sg/news/singapore/travellers-false-declarations-can-be-prosecuted-moh and https://www.channelnewsasia.com/news/singapore/covid19-coronavirus-charged-false-info-moh-contact-tracing-12471894

Reply:
Added on [the form](https://docs.google.com/forms/u/2/d/e/1FAIpQLScGBotz1GKpaTqvZEAw7HecUSSGfBhbEEVxKeTP0kuPSURSAA/viewform).

Reply:
> All collected information will be destroyed when the authorities lower the DORSCON level to Yellow.

Suggest changing this to 
> All collected information will be destroyed after 90 days or when the authorities deem that contacting tracing is no longer needed, whichever is earlier.

Lowering to Yellow may not reduce the authorities' want for contact tracing. After all, DORSCON could be lowered to Yellow even when the spread is still ongoing. https://www.channelnewsasia.com/news/singapore/dorscon-level-covid19-transmits-worldwide-coronavirus-12469484. As for 90 days limit, it is a time limit that I think is just a bit longer than what authorities would ask for given that the incubation is up to 14 days (some say 20-30+ days), and with allowance for backfilling, maybe 2 months of prior data?

Reply:
also name field to be compulsory

Reply:
Thanks, changed.